docs/youtube/livestream.md

```diff
@@ -0,0 +1,3 @@
+# Moderating Youtube
+
+Coming soon! :)
```

Comment:
Probably better to delete the link to this file, rather than create a file

Reply:
I'll do it.

Reply:
Thanks 👍